hydra-main/src/main/java/com/addthis/hydra/job/web/resources/JobsResource.java

```diff
@@ -1372,14 +1373,10 @@ private static Response buildServerError(Exception exception) {
         if (message == null) {
             message = exception.toString();
         }
-
-        String stack = Throwables.getStackTraceAsString(exception);
-
         final String response = "{" +
-                "\"error\": \"A java exception was thrown." +
-                "\"message\": \"" + StringEscapeUtils.escapeEcmaScript(message) + "\"" +
-                "\"stack\": \"" + StringEscapeUtils.escapeEcmaScript(stack) + "\"" +
-                "\"}";
+                                "\"error\": \"A java exception was thrown." +
+                                "\"message\": \"" + StringEscapeUtils.escapeEcmaScript(message) + "\"" +
+                                "\"}";
```

Comment:
Hmm... Is the intent to respond with json text? This is not valid json (closing quote for the error message text is missing, so is the comma). This looks to be an existing bug, but we should fix regardless.